tests/h/views/activity_test.py

```diff
@@ -208,6 +225,46 @@ def search(self, patch):
         return search
 
 
+@pytest.mark.usefixtures('routes')
+class TestGroupSearchMoreInfo(object):
+
+    def test_it_redirects_to_group_search(self, pyramid_request):
+        """It should redirect and preserve the search query param."""
+        pyramid_request.matchdict['pubid'] = 'test_pubid'
+        pyramid_request.POST = {'q': 'foo bar'}
```

Comment:
It actually doesn't make any difference in the test. The `more_info` param is only used by Pyramid to select which view callable to call, the view callable itself doesn't actually use it. And in the test, we don't use Pyramid's view selection mechanism, we just call the view function directly.

But it makes the test more realistic so I'll add it anyway